IssueTracker/DataAccess/Repository/FileRepository.cs

```diff
@@ -0,0 +1,43 @@
+﻿using DataAccess.DbAccess;
+
+namespace DataAccess.Repository;
+public class FileRepository : IFileRepository
+{
+    private readonly ISQLDataAccess _db;
+    public FileRepository(ISQLDataAccess db)
+    {
+        _db = db;
+    }
+    public async Task<string?> AddAsync(Models.File entity)
```

Comment:
What's the use case when the response can be null?

Reply:
so that it knows it didn't save in the db